priv/templates/enum.ex.eex

```diff
@@ -4,6 +4,8 @@ defmodule <%= @module %> do
   <% end %>
   use Protobuf, <%= @use_options %>
 
+  def full_name, do: "<%= @full_name %>"
```

Comment:
Takes care of the quotes and avoids surprises by always returning a valid string.

```suggestion
  def full_name, do: <%= inspect(@full_name) %>
```